app/views/team_users/update.js.erb

```diff
@@ -1,6 +1,12 @@
 <% if team_user.errors.empty? %>
-  $("#team_user_<%= team_user.id %> td div.role").html("<%= team_user.role %>");
-  $("#change_role_team_user_<%= team_user.id %>").fadeToggle();
+  $("#team_user_<%= team_user.id %> td div.role").html("<%= team_user.role.capitalize %>");
+  $("#team_user_<%= team_user.id %> td .btn-edit-role i").removeClass('fa-close');
+  $("#team_user_<%= team_user.id %> td .btn-edit-role i").addClass('fa-pencil');
+  $("#change_role_team_user_<%= team_user.id %>").fadeToggle(function() {
+    $("#team_user_<%= team_user.id %> td div.role").fadeToggle();
+  });
+  el.removeClass('fa-close')
```

Comment:
I see it defined in app/assets/javascript/teams.coffee .. but not sure if we speak about the same var